source/_integrations/reolink.markdown

```diff
@@ -23,7 +23,7 @@ A Reolink user account with admin privileges is needed for the proper operation
 {% include integrations/option_flow.md %}
 {% configuration_basic %}
 Protocol:
-description: Switch between RTSP, RTMP or FLV streaming protocol.
+  description: Switch between RTSP, RTMP or FLV streaming protocol.
```

Comment:
Another option would be this if the bullet list style is wanted
```suggestion
  - description: Switch between RTSP, RTMP or FLV streaming protocol.
```

Reply:
A dash will cause the CI tests to fail. I looked up another example to verify.